etc/nebula-graphd.conf.production

```diff
@@ -55,4 +55,6 @@
 
 ########## authorization ##########
 # Enable authorization
---enable_authorize=false
\ No newline at end of file
+--enable_authorize=false
+# Authentication type
+--auth_type=password
```

Comment:
It's better to add some comments  here. right ?

Reply:
Good catch